test/functional/wallet_importmulti.py

```diff
@@ -632,5 +639,52 @@ def run_test (self):
         result = self.nodes[1].getaddressinfo(address["address"])
         assert_equal(result['ismine'], True)
 
+        # Import pubkeys with key origin info
+        self.log.info("Addresses should have hd keypath and master key id after import with key origin")
+        pub_addr = self.nodes[1].getnewaddress()
+        pub_addr = self.nodes[1].getnewaddress()
```

Comment:
Why call twice? Is it just to have a non-zero keypath index? Perhaps add comment if deliberate

Reply:
Added a comment.